src/sentry/models/organizationmember.py

```diff
@@ -239,6 +239,7 @@ def legacy_token(self):
     def generate_token(self):
         return uuid4().hex + uuid4().hex
 
+    # TODO: does this need to be updated?
```

Comment:
Ideally yes. New membership invites can start using organization slugs now.

Reply:
i'll open another PR to address this